build-tools/packages/build-tools/src/repoPolicyCheck/repoPolicyCheck.ts

```diff
@@ -149,18 +149,20 @@ lineReader.on("line", (line) => {
 });
 
 lineReader.once("close", () => {
-    handlers.forEach((h) => {
-        const final = h.final;
-        if (final) {
-            const result = runWithPerf(h.name, "final", () =>
-                final(pathToGitRoot, program.resolve),
-            );
-            if (result?.error) {
-                process.exitCode = 1;
-                console.log(result.error);
+    handlers
```

Comment:
Not the right place for this fix. Old build tools are being deprecated. New entry point is in flub: https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/src/commands/check/policy.ts

Reply:
`<lurks back to vacation>`

Reply:
I'll look into the flub side of things. At a glance, seems to me like we should process the `--handler` flag earlier so we can filter ``handlersToRun` from the very start? That would also fix reporting which handlers are going to run when `--listHandlers` is specified; right now if combining those two, we'd print all handlers instead of just those matching the regex.